e2e/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test:dev": "cypress open",
-    "test": "cypress run",
```

Comment:
Why was this no required before?

Reply:
that's for local development, forgot to delete it, thanks for catching!